R/zchunk_L210.resources.R

```diff
@@ -56,28 +62,44 @@ module_energy_L210.resources_DISABLED <- function(command, ...) {
              "L210.GrdRenewRsrcCurves_tradbio",
              "L210.GrdRenewRsrcMax_tradbio",
              "L210.DepRsrcTechChange_SSP1",
-             "L210.DepRsrcEnvironCost_SSP1",
+             "L210.DepRsrcEnvironCost_SSP1", # Units
              "L210.DepRsrcTechChange_SSP2",
-             "L210.DepRsrcEnvironCost_SSP2",
+             "L210.DepRsrcEnvironCost_SSP2", # Units
              "L210.DepRsrcTechChange_SSP3",
-             "L210.DepRsrcEnvironCost_SSP3",
+             "L210.DepRsrcEnvironCost_SSP3", # Units
              "L210.DepRsrcTechChange_SSP4",
-             "L210.DepRsrcEnvironCost_SSP4",
+             "L210.DepRsrcEnvironCost_SSP4", # Units
              "L210.DepRsrcTechChange_SSP5",
-             "L210.DepRsrcEnvironCost_SSP5"))
+             "L210.DepRsrcEnvironCost_SSP5")) # Units
   } else if(command == driver.MAKE) {
 
+    # Silence package checks
+    SSP <- available <- cal.production <- capacity.factor <- curve.exponent <- depresource <- environCost <-
+      extractioncost <- fuel <- gdpSupplyElast <- grade <- market <- maxSubResource <- mid.price <- object <-
+      `output-unit` <- `price-unit` <- region <- resource <- resource_type <- scenario <-
+      subResourceCapacityFactor <- subresource <- subresource_type <- value <- year <- NULL
+
     all_data <- list(...)[[1]]
 
     # Load required inputs
     GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
     A_regions <- get_data(all_data, "energy/A_regions")
-    A10.rsrc_info <- get_data(all_data, "energy/A10.rsrc_info")
+    A10.rsrc_info <- get_data(all_data, "energy/A10.rsrc_info") %>%
+      gather(year, value, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
```

Comment:
Replace these new lines with call to new `gather_year` function ( #821 ). This also applies to all `gather %>% mutate(year = as.integer(year))` sequences below.